src/community/default.ts

```diff
@@ -67,31 +67,35 @@ let _defaultCommunity: Community|undefined
  * @internal
  */
 export const _getDefault = async (repo?:Repo): Promise<Community> => {
-    if (repo == undefined) {
-        repo = new Repo("default")
-        await repo.init({})
-        await repo.open()
-    }
-
-    if (_defaultCommunity !== undefined) {
-        return _defaultCommunity.start()
-    }
-
-    const node = await p2p.createNode({ bootstrapAddresses: defaultNotaryGroup.getBootstrapAddressesList() });
-    node.on('error', (err: Error) => {
-        console.error('p2p error: ', err)
+    return new Promise(async (resolve,reject) => {
+        if (repo == undefined) {
+            repo = new Repo("default")
+            await repo.init({})
+            await repo.open()
+        }
+    
+        if (_defaultCommunity !== undefined) {
```

Comment:
I don't know if there is a TypeScript idiom / difference here compared to vanilla JS, but should we standardize on `===` & `!==` or `==` & `!=`? Looks like we're using a combination of `==` & `!==` currently (at least in this code).

Reply:
yeah should be === and !== I think? I just change it when typescript complains :)